ChangeLog.d/c99.txt

```diff
@@ -0,0 +1,3 @@
+Changes
+   * Pass standard c99 to compiler in CMakeLists.txt
```

Comment:
I have no idea what “Pass standard c99 to compiler” is supposed to mean. I don't think this requires a changelog entry, but if you think it does, the changelog entry needs to be comprehensible.

Reply:
Changelog entry has been removed.